packages/react-dev-utils/webpackHotDevClient.js

```diff
@@ -272,8 +272,7 @@ function tryApplyUpdates(onHotUpdateSuccess) {
     return;
   }
 
-  // https://webpack.github.io/docs/hot-module-replacement.html#check
-  module.hot.check(/* autoApply */true, function(err, updatedModules) {
+  var checkCallback = function(err, updatedModules) {
```

Comment:
Minor nit: let’s also make this a regular function declaration, i.e. `function handleApplyUpdates(...)`